e2e/test/scenarios/sharing/reproductions/26988-embedding-dynamic-settings.cy.spec.js

```diff
@@ -40,7 +42,7 @@ describeEE("issue 26988", () => {
       acceptTerms: false,
     });
 
-    cy.wait("@dashboard");
+    cy.wait("@previewDashboard");
```

Comment:
The preview didn't seem to work in Cypress before. I've fixed that in `frontend/src/metabase/lib/dom.js`